### ISSUE TITLE
Installing Powershell 5 on Windows 7 breaks Nuget Visual Studio package manager consoles (2013/2015)

### ISSUE THREAD

**Issue by tmulkern** [NuGet/Home#3165]
If you install Powershell 5, specifically Win7AndW2K8R2-KB3134760-x64.msu downloaded from here https://www.microsoft.com/en-us/download/details.aspx?id=50395, the following will be displayed in Package Manager Console for both Visual Studio 2013 and Visual Studio 2015

![image](https://cloud.githubusercontent.com/assets/8748428/16952373/dc17c8f2-4dc0-11e6-92c3-1726738f6ffd.png)

The Package Manager Console will not work, the fix of course is to remove Powershell 5.

**Comment by emgarten** [comment]
Related: https://github.com/NuGet/Home/issues/2836

**Comment by tmulkern** [comment]
@emgarten Thanks

**Comment by PatoBeltran** [comment]
Closing as https://github.com/NuGet/Home/issues/2836 provides a workaround

**Comment by nvrnight** [comment]
None of the workarounds provided works for me. I have the same problem, except the location it is looking for ps1xml files is different. it's searching in my local solution directory for these files for some reason. This is only after I installed Powershell v5.

Error when opening Package Management Console is:
Could not find file 'C:\path\to\my\solution\GetEvent.types.ps1xml'.
Could not find file 'C:\path\to\my\solution\FileSystem.format.ps1xml'.
Could not find file 'C:\path\to\my\solution\FileSystem.format.ps1xml'.

EDIT 1:
When opening VS without my solution it checks for the following locations:
Could not find file 'C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\GetEvent.types.ps1xml'.
Could not find file 'C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\FileSystem.format.ps1xml'.
Could not find file 'C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\FileSystem.format.ps1xml'.

EDIT 2:
Fixed it(Just kidding, see EDIT 3) by using the similar workaround posted in the other thread but for the location it's looking for above instead. copy C:\windows\System32\WindowsPowerShell\v1.0\*ps1xml "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\"

EDIT 3:
Unfortunately the fix above only worked when a solution isn't opened. It is still looking for these files in the local solution directory for some reason. I could copy these files to my solution directory to get it working of course, but this seems to be really unnecessary to have to do this for every project I work on.
